codex-ui/src/vue/components/tabbar/Tab.vue

```diff
@@ -64,12 +67,17 @@ withDefaults(
      * Name of the icon to be diplayed in the left slot, else undefined
      */
     icon?: string;
+
+    /**
+     * When user clicks on tab he will be redirect according to this url
+     */
+    url: string;
```

Comment:
where url is used?

Reply:
on deletion of the tab (when we delete tab we need to know it's url)